# Goal
geometry_msgs/PoseStamped target
---
# Result
geometry_msgs/PoseStamped next_waypoint
bool goal_reached
---
# Feedback
float32 remaining_distance
geometry_msgs/Vector3 f_attr
geometry_msgs/Vector3 f_rep
string debug_info
